<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>293</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CLICalculator</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="9,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="listWidget_history">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(45, 45, 45);
color:rgb(249, 145, 87);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="autoScroll">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="selectionRectVisible">
          <bool>true</bool>
         </property>
         <property name="itemAlignment">
          <set>Qt::AlignLeading</set>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>2+2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>= 4</string>
          </property>
          <property name="background">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>153</red>
             <green>204</green>
             <blue>253</blue>
            </color>
           </brush>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_time">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(45, 45, 45);
color:rgb(102, 204, 204)</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionRectVisible">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>18:30:50</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>2000</width>
         <height>2000</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(57, 57, 57);
color:rgb(249, 145, 87);</string>
       </property>
       <property name="text">
        <string>2+2</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="cursorPosition">
        <number>3</number>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>342</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(152, 204, 60);
color:rgb(255, 255, 255);</string>
   </property>
   <widget class="QMenu" name="menuSession">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Session</string>
    </property>
    <addaction name="actionExport_session"/>
    <addaction name="actionImport_session"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="actionImport"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionClear_history"/>
    <addaction name="actionClear_input"/>
    <addaction name="actionClear_all"/>
    <addaction name="separator"/>
    <addaction name="actionInsert_Previous_Input"/>
    <addaction name="actionInsert_Next_Input"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionShow_time_stamps"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuSession"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExport_session">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
  <action name="actionImport_session">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="actionClear_history">
   <property name="text">
    <string>Clear History</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionClear_input">
   <property name="text">
    <string>Clear Input</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionClear_all">
   <property name="text">
    <string>Clear All</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionShow_time_stamps">
   <property name="text">
    <string>Toggle Time Stamps</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionInsert_Previous_Input">
   <property name="text">
    <string>Insert Previous Input</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="actionInsert_Next_Input">
   <property name="text">
    <string>Insert Next Input</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>listWidget_history</tabstop>
  <tabstop>listWidget_time</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
